internal/server/handlers.go

```diff
@@ -329,6 +329,7 @@ func (a *API) CreateToken(c *gin.Context, r *api.EmptyRequest) (*api.CreateToken
 	if access.AuthenticatedIdentity(c) != nil {
 		err := a.UpdateIdentityInfoFromProvider(c)
 		if err != nil {
+			// TODO: why would this fail? seems like this should be a 5xx error
```

Comment:
I added the missing timeout in #2356.

I realized this is only the `CreateToken` endpoint, so the CLI never calls this directly. I'm not sure how the kubectl CLI handles the different errors. It likely just echo's back the response body. There will be more work to separate out the "user was deleted" error from other errors, so maybe we can do that in a follow up?

I don't think we can simply say "all other errors are unauthorized", because if the error is the DB not being available, it should still be a 500.